AdguardTeam/FiltersRegistry
# update compiler to v1.1.107

How to check:
1. add to `filters_3_Spyware / template.txt`
```
!-------------------------------------------------------------------------------!
!-------------- CNAME mail trackers --------------------------------------------!
!-------------------------------------------------------------------------------!
@include "https://raw.githubusercontent.com/AdguardTeam/cname-trackers/master/data/combined_disguised_mail_trackers_justdomains.txt" /addModifiers="image"
```
(could be added different modifiers ex. `/addModifiers="script,redirect=noopjs"`)
2. run `node index.js`
3. check that `filters_3_Spyware / filters.txt` domains with `$image` modifier appear.


- - -

https://github.com/AdguardTeam/FiltersCompiler/blob/master/CHANGELOG.md
